### ISSUE TITLE
Not getting desired output

### ISSUE THREAD

**Issue by Mary206** [singhmurari96/Fake-News-Detection-Using-Deep-Learning#1]
# Sequential Model
model = Sequential()

# embeddidng layer
model.add(Embedding(total_words, output_dim = 240))

# Bi-Directional RNN and LSTM
model.add(Bidirectional(LSTM(128)))

# Dense layers
model.add(Dense(128, activation = 'relu'))
model.add(Dense(1,activation= 'sigmoid'))
model.compile(optimizer='adam', loss='binary_crossentropy', metrics=['acc'])
model.summary() 
 This part of code is not getting desired output. Can you please check with it.
<img width="441" alt="error" src="https://github.com/singhmurari96/Fake-News-Detection-Using-Deep-Learning/assets/83813266/f03f8329-077e-4056-95d0-5e0e90832f50">
